test/systemtest/conftest.py

```diff
@@ -125,3 +126,8 @@ def spiflash(pytestconfig):
     path = Path(pytestconfig.getoption('spiflash')).resolve()
     assert path.is_file()
     return path
+
+@pytest.fixture(scope="session")
+def logfile(pytestconfig):
+    """Return path to logfile."""
+    return pytestconfig.getoption('log')
```

Comment:
All the other fixtures return a Path; I'd suggest doing something like 
```python
log = pytestconfig.getoption('log')
if log == '':
  return
path = Path(log).resolve()
assert not path.is_directory()
return path
```

Reply:
fixed, thanks for the suggestion.